appeals/web/src/server/appeals/appeal-details/linked-appeals/add/add.controller.js

```diff
@@ -111,7 +111,9 @@ export function renderLeadAppeal(request, response) {
 	const pageContent = changeLeadAppealPage(
 		currentAppeal,
 		session.linkableAppeal.linkableAppealSummary,
-		session.linkableAppeal.leadAppeal
+		session.linkableAppeal.leadAppeal,
+		query.backUrl ? String(query.backUrl) : undefined,
```

Comment:
There's a new utility we've recently added to handle back link queries in a consistent way, it would be good to use that here if possible: https://github.com/Planning-Inspectorate/appeals-back-office/blob/main/appeals/web/src/server/lib/url-utilities.js#L62